core/components/molecules/dialog/dialog.story.js

```diff
@@ -13,10 +13,11 @@ storiesOf('Dialog').add('default', () => (
     <Dialog
       open
       title="Example Dialog"
-      onClose={() => {}}
+      role="destructive"
+      onClose={() => { }}
       actions={[
-        new Dialog.Action('OK', () => {}, 'primary'),
-        new Dialog.Action('Cancel', () => {})
+        new Dialog.Action('OK', () => { }, 'primary'),
```

Comment:
Not for this PR, but @francocorreasosa Do you think we should replace this with just buttons™

Reply:
Of course 😉